static/css/character-selection.css

```
@@ -0,0 +1,691 @@
+    transform: translateY(-4px);
+}
+
+.character-card.selected::before {
+    opacity: 1;
+}
+
+.card-icon {
+    font-size: 28px;
+    margin-bottom: 16px;
+    filter: drop-shadow(0 2px 8px rgba(0, 0, 0, 0.3));
+    font-family: "Monaco", "Courier New", monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)